app/src/features/sessionBook/screens/SavedSessionScreen/components/SessionViewerBottomSheet/components/SessionInfo/SessionInfo.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useCallback, useMemo } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { CgAlignCenter } from "@react-icons/all-files/cg/CgAlignCenter";
+import { MdOutlineViewHeadline } from "@react-icons/all-files/md/MdOutlineViewHeadline";
+import { IoIosGlobe } from "@react-icons/all-files/io/IoIosGlobe";
+import { IoMdTime } from "@react-icons/all-files/io/IoMdTime";
+import { MdToday } from "@react-icons/all-files/md/MdToday";
+import { InlineInput } from "componentsV2/InlineInput/InlineInput";
+import {
+  getSessionRecordingAttributes,
+  getSessionRecordingId,
+  getSessionRecordingMetaData,
+} from "store/features/session-recording/selectors";
+import { epochToDateAndTimeString, msToHoursMinutesAndSeconds } from "utils/DateTimeUtils";
+import { sessionRecordingActions } from "store/features/session-recording/slice";
+import { updateSessionDescription, updateSessionName } from "../../../utils";
+import { getUserAuthDetails } from "store/selectors";
+import "./sessionInfo.scss";
+
+export const SessionInfo: React.FC = () => {
+  const dispatch = useDispatch();
+  const user = useSelector(getUserAuthDetails);
+  const recordingId = useSelector(getSessionRecordingId);
+  const sessionMetadata = useSelector(getSessionRecordingMetaData);
+  const sessionAttributes = useSelector(getSessionRecordingAttributes);
+
+  const handleSessionNameUpdate = useCallback(() => {
+    if (recordingId && sessionMetadata?.name) {
+      updateSessionName(user?.details?.profile?.uid, recordingId, sessionMetadata.name);
+    }
+  }, [recordingId, sessionMetadata?.name, user?.details?.profile?.uid]);
+
+  const handleSessionDescriptionUpdate = useCallback(() => {
+    if (recordingId && sessionMetadata?.description) {
```

Comment:
Confirm this if we need analytics over here or not.

Reply:
An event is being already sent inside `updateSessionName` funtion